src/v/kafka/server/quota_manager.cc

```diff
@@ -72,6 +74,42 @@ quota_manager::maybe_add_and_retrieve_quota(
     return it;
 }
 
+void quota_manager::initialize_target_tp_rate_per_client_from_config() {
+    const std::vector<ss::sstring> clients_with_limit
+      = _target_tp_rate_per_client_group_config();
+    for (const auto& client_and_limit : clients_with_limit) {
+        auto parsing_setting = config::parse_client_group_byte_rate_quota(
+          client_and_limit);
+        _target_tp_rate_per_client_group.insert(parsing_setting.value());
+    }
+}
+
+std::optional<std::string_view> quota_manager::get_client_quota(
```

Comment:
This is called get client quota but returns client id

Reply:
If client has no group then quota_id is equal to client_id
If client has group then quota_id is group_name

Reply:
Maybe a more suitable name would be `get_client_quota_id` then?